File: Java/BeeCrowd/BeeCrowd1052.java
import java.util.Scanner;

/**
 * Leia um valor inteiro entre 1 e 12, inclusive. Correspondente a este valor, deve ser apresentado como resposta o mês do ano por extenso, 
 * em inglês, com a primeira letra maiúscula.
 *      e.x: 4 -> April
 *
 * @author Matheus Maia
 * @version 20/08/2022
 */

class BeeCrowd1052 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        String[] meses = {"January", "February", "March", "April", "May", "June",
        "July", "August", "September", "October", "November", "December"};

        int entrada = in.nextInt();
        in.close();

        System.out.println(meses[entrada - 1]);
    }
}

File: Python/BeeCrowd1052.py
# Beecrowd1052:
#
# Leia um valor inteiro entre 1 e 12, inclusive. Correspondente a este valor, deve ser 
# apresentado como resposta o mês do ano por extenso, em inglês, com a primeira letra maiúscula.
#
# Ex: 4 -> April


meses = ['January', 'February', 'March', 'April', 'May', 'June',
'July', 'August', 'September', 'October', 'November', 'December']

entrada = int(input())

print(f'{meses[entrada - 1]}\n')
